### PR TITLE
BIC-174 # Errors from server persisted to pending queue. Forms Error …

### PR DESCRIPTION
**Pull request by simonmarklar**
…format updated to v3. Documentation for dealing with server side errors added to pending queue docuemtation

### CODE REVIEW

File: src/bic/view-form-controls.js [hunk 1]
```diff
@@ -19,8 +18,11 @@ define(function (require) {
   var MODEL_STATUS = require('bic/enum-model-status');
   var uiTools = require('bic/lib/ui-tools');
 
+
   // this module
+  var FormControlView;
```
Comment (jokeyrhyme) [inline src/bic/view-form-controls.js]: Seems to be another vestigial assign-variable-then-return-it pattern here. :)

Comment (simonmarklar) [inline src/bic/view-form-controls.js]: :+1:  removed it

**Comment by jokeyrhyme** [comment]
LGTM :+1:

**Comment by jokeyrhyme** [comment]
Just waiting to get the all clear for https://github.com/blinkmobile/forms/pull/4

**Pull request merged**